addons/docs/src/frameworks/web-components/custom-elements.ts

```diff
@@ -50,30 +50,57 @@ function mapData(data: TagItem[], category: string) {
   return (
     data &&
     data
-      .filter((item) => !!item)
+      .filter((item) => item && item.name)
       .reduce((acc, item) => {
         if (item.kind === 'method') return acc;
 
         const type =
           category === 'properties' ? { name: item.type?.text || item.type } : { name: 'void' };
-        acc[item.name] = {
-          name: item.name,
-          required: false,
-          description: item.description,
-          type,
-          table: {
-            category,
-            type: { summary: item.type?.text || item.type },
-            defaultValue: {
-              summary: item.default !== undefined ? item.default : item.defaultValue,
-            },
-          },
-        };
+
+        switch (category) {
+          case 'events':
+            // eslint-disable-next-line no-return-assign
```

Comment:
If you use curlie braces here instead of parens you shouldn’t need the lint ignore, IIUC.

Reply:
I believe you need the `except-parens` config for that, but my guess is that that isn't in the config (vscode keeps complaining if I remove the comment)